symcrypt-sys/build.rs

```diff
@@ -0,0 +1,90 @@
+extern crate bindgen;
+
+use std::env;
+use std::path::PathBuf;
+
+fn main() {
+    #[cfg(target_os = "windows")] 
+    {
+        println!("cargo:rustc-link-search=native=C:/Windows/System32/"); // ! Work around, looking for better solution
+        println!("cargo:libdir=./inc");
+        println!("cargo:rustc-link-lib=dylib=symcrypttestmodule"); // test module used in lieu of official symcrypt dll
+        // this dll will be in Windows/System32. This is to mirror future plans; as symcrypt is planned to ship with windows
+        // symcrypttestmodule* files must be placed in Windows/System32/ as a workaround at the moment.
+    }
+    
+    #[cfg(target_os = "linux")]
+    {   
+        println!("cargo:libdir=./inc");
+        println!("cargo:rustc-link-lib=dylib=symcrypt"); // the lib prefix for libsymcrypt is implied on linux
+
+        // TODO: Create a script that copies all libsymcrypt.so* files from SymCrypt path to /lib/x86_64-linux-gnu/
+        // The ld linker will look for the symcrypt.so files within /lib/x86_64-linux-gnu/. No need to set a hardcoded path.

```

Comment:
These files will already be present on distros that ship with SymCrypt, such as Mariner. This would still be necessary on Ubuntu though, unless we create SymCrypt packages for distros like Ubuntu. That's not currently on the roadmap, though.

Reply:
Noted, I will keep that in mind when writing the instillation instructions later down the line